bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -1549,6 +1554,60 @@ Mapping method from Nitrogen deposition input file to the model resolution
     copy     = copy using the same indices
 </entry>
 
+<!-- ========================================================================================  -->
+<!-- FAN Namelist (only used when bgc=cn/cndv and use_fan=.true.)                 -->
+<!-- ========================================================================================  -->
+
+<entry id="stream_year_first_fan" type="integer" category="datasets"
+       group="fan_nml" valid_values="" >
+First year to loop over for FAN Nitrogen (manure) Deposition data
+</entry>
+
+<entry id="stream_year_last_fan" type="integer" category="datasets"
+       group="fan_nml" valid_values="" >
+Last year to loop over for FAN Nitrogen (manure) Deposition data
+</entry>
+
+<entry id="model_year_align_fan" type="integer" category="datasets"
+       group="fan_nml" valid_values="" >
+Simulation year that aligns with stream_year_first_ndep2 value
+</entry>
+
+<entry id="stream_fldfilename_fan" type="char*256" category="datasets"
+       input_pathname="abs" group="fan_nml" valid_values="" >
+Filename of input stream data for FAN Nitrogen (manure) Deposition
+</entry>
+
+<entry id="fan_mapalgo" type="char*256" category="datasets"
+       group="fan_nml" valid_values="bilinear,nn,nnoni,nnonj,spval,copy" >
+Mapping method from FAN Nitrogen (manure) deposition input file to the model resolution
+    bilinear = bilinear interpolation
+    nn       = nearest neighbor
+    nnoni    = nearest neighbor on the "i" (longitude) axis
+    nnonj    = nearest neighbor on the "j" (latitude) axis
+    spval    = set to special value
+    copy     = copy using the same indices
+</entry>
+
+<entry id="fan_to_bgc_crop" type="logical" category="physics"
+       group="fan_nml" valid_values="" >
+Toggle to connect the FAN N pools to soil biogeochemistry for CROP VEGETATION columns
+</entry>
+
+<entry id="fan_to_bgc_veg" type="logical" category="physics"
```

Comment:
It's not clear from the description here how this is different from 'fract_spread_grass', below?  Eventually this would be used to connect to soil BGC to FAN fluxes on 'pasture' columns?

Reply:
I'll fix the description. Fract_spread_grass transfers manure N from crop to native veg. columns, so that fraction is not available for crops. Fan_to_bgc_veg then determines what happens to that N after losses in FAN.  

Originally, we had the idea of having some of the manure spread continuously on grasslands, and a the remaining part spread on crops together with the fertilizer. However, it was hard to define those fractions, and comparison with atmospheric measurements indicated that already the 20-day fertilization overshoots the peak observed from springtime fertilization. 

In its current form, fract_spread_grass has quite little effect on the NH3 emisisons. It adds some complexity to the code, and you may want to consider whether to keep it or not. There could be areas in world where fract_spread_grass = 0 would result in unrealistic N inputs to the crop columns, but I don't know if this a real problem or not.